pug/page-contents/icons_content.html

```diff
@@ -3,7 +3,7 @@
     <div class="col s12 m8 offset-m1 xl7 offset-xl1">
 
     <div id="usage" class="scrollspy">
-            <p class="caption">We have included 932 Material Design Icons courtesy of Google. You can download them directly from the <a href="https://google.github.io/material-design-icons/#icon-font-for-the-web">Material Design specs</a>.</p>
+            <p class="caption">We use Google Material Icons by Google. They provide a searchable list here (which we do not include in the documentation here), which will also show you the relevant icon names for the CSS classes. You can download the icons directly from the <a href="https://google.github.io/material-design-icons/#icon-font-for-the-web">Material Design specs</a>.</p>
```

Comment:
```suggestion
            <p class="caption">We use Google Material Icons by Google. They provide a searchable list (which we do not include in the documentation here), which will also show you the relevant icon names for the CSS classes. You can download the icons directly from the <a href="https://google.github.io/material-design-icons/#icon-font-for-the-web">Material Design specs</a>.</p>
```

Reply:
Committed the suggested changes please let me know if there is any other change.

Regards.